ncc/client/client.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2012 Google Inc. All Rights Reserved.
```

Comment:
I don't see spliting the file of ncc_client.go is necessary for your purpose. could you just expose the dummy one in dummy.go without changing other things?

Reply:
Done.